cmd/pitr/recoverer/recoverer.go

```diff
@@ -136,16 +137,27 @@ func getStartGTIDSet(c BackupS3) (string, error) {
 	if err != nil {
 		return "", errors.Wrap(err, "new storage manager")
 	}
+	listInfo, err := s3.ListObjects("xtrabackup_info")
+	if err != nil {
+		return "", errors.Wrapf(err, "list %s info fies", prefix)
+	}
+	sort.Strings(listInfo)
 
-	infoObj, err := s3.GetObject("xtrabackup_info.00000000000000000000") //TODO: work with compressed file
+	var lastGTID string
+	if len(listInfo) == 0 {
```

Comment:
worth to put it before sort =)

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/743/commits/55509d9d8f98677a13f4c5d3b91c3487c48ed868